src/ecalc/libraries/libecalc/fixtures/libecalc/fixtures/cases/consumer_system_v2/data/consumer_system_v2.yaml

```diff
@@ -26,57 +29,107 @@ INSTALLATIONS:
   - NAME: installation
     HCEXPORT: 17
     FUEL: fuel_gas
-    FUELCONSUMERS:
-      - NAME: compressor_system_v2
-        CATEGORY: COMPRESSOR
-        TYPE: COMPRESSOR_SYSTEM@v2
+    CATEGORY: FIXED
+    GENERATORSETS:
```

Comment:
I do not understand why you change to only test el-consumers/el-consumer-system and removed the old to test fuel consumers? Why not both?

Reply:
I've reverted to test both now.